zeebe/engine/src/main/java/io/camunda/zeebe/engine/state/appliers/MappingDeletedApplier.java

```diff
@@ -45,31 +35,8 @@ public void applyState(final long key, final MappingRecord value) {
               "Expected to delete mapping with key '%s', but a mapping with this key does not exist.",
               value.getMappingKey()));
     }
-    final var persistedMapping = mapping.get();
-    removeMappingFromRoleState(persistedMapping);
-    removeMappingFromGroupState(persistedMapping);
-    removeMappingFromTenantState(persistedMapping);
     // remove mapping from authorization state
     authorizationState.deleteOwnerTypeByKey(mappingKey);
```

Comment:
❓ Can you double check if here we missed to call `deleteAuthorizationsByOwnerKeyPrefix()` ?

Reply:
Good catch, I think so :+1:

Reply:
Fixed in 5949c0d